### PR TITLE
[Merged by Bors] - Add toledo support

### PR DESCRIPTION
**Pull request by paulhauner**
## Issue Addressed

NA

## Proposed Changes

Adds support for the [Toledo](https://github.com/protolambda/toledo) dev-net.

```bash
lighthouse --testnet toledo bn --http
```

This is for development only, we do not recommend users to join this testnet.

## Additional Info

- ~~Blocked on #1862~~

### CODE REVIEW

File: common/eth2_testnet_config/built_in_testnet_configs/toledo/config.yaml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+MAX_VALIDATORS_PER_COMMITTEE: 2048
+# 2**2 (= 4)
+MIN_PER_EPOCH_CHURN_LIMIT: 4
+# 2**16 (= 65,536)
+CHURN_LIMIT_QUOTIENT: 65536
+# See issue 563
+SHUFFLE_ROUND_COUNT: 90
+# `2**14` (= 16,384)
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 16384
+# Dec 1, 2020, 12pm UTC
```
Comment (pawanjay176) [inline common/eth2_testnet_config/built_in_testnet_configs/toledo/config.yaml:21]:
```suggestion
# Nov 10, 2020, 12pm UTC
```

**Review by pawanjay176 — APPROVED** [review]
Tested this out locally. Works fine, finding peers and waiting for genesis :+1: 
LGTM! Just a minor typo in the config comment.

**Review by michaelsproul — APPROVED** [review]
Looks great!

I found a bunch of peers instantly, so I'm happy with that

**Comment by paulhauner** [comment]
Thanks @michaelsproul and @pawanjay176!

bors r+

**Comment by michaelsproul** [comment]
bors retry

**Pull request closed without merging**